### ISSUE TITLE
Cannot create new project, Godot crashes

### ISSUE THREAD

**Issue by Zylann** [godotengine/godot#10569]
Windows 10 64 bits, master

I created a project fine with this build, but after that, any project I tried to create since have failed into a crash. I'm unsure wether it's reproducible for someone else yet, but for me it's stuck on repro 100%.

basically:

1) Launch Godot with debugger attached (MSVC for me), with no working dir and no options:

![image](https://user-images.githubusercontent.com/1311555/29588357-7687c274-8791-11e7-8794-ff2225da963e.png)

2) Choose "Create project", give it a new folder somewhere (on my end, it starts in C:/ drive, I changed it to D:/ and chose some folder to create the project in)
3) Validate, the project manager closes (and the debugger detaches so can't debug passed this point :( )
4) Godot crashes in the middle of splash screen.

Trying to open the project later on works fine.

Full log of the steps:

```
Activated GL 3.3 contextOpenGL ES 3.0 Renderer: GeForce 940M/PCIe/SSE2
GLES3: max ubo light: 409
GLES3: max ubo reflections: 455, ubo size: 144
ERROR: ClassDB::bind_methodfi: Couldn't bind method 'set_expand_margin_all' for instance: StyleBoxFlat
   At: core\class_db.cpp:1182
ERROR: ClassDB::bind_methodfi: Method/Function Failed, returning: 0
   At: core\class_db.cpp:1184
ERROR: ClassDB::bind_methodfi: Couldn't bind method 'set_expand_margin_individual' for instance: StyleBoxFlat
   At: core\class_db.cpp:1182
ERROR: ClassDB::bind_methodfi: Method/Function Failed, returning: 0
   At: core\class_db.cpp:1184
mode: spatial
1
SVG_GENERATION TIME: 0.251
vertex uses function: unpack_normal
fragment uses function: unpack_normal
OPENING: D:/PROJETS/INFO/GODOT/TESTS/3/lol/OpenGLErrors5 (D:::PROJETS::INFO::GODOT::TESTS::3::lol::OpenGLErrors5)
running cmdline: "D:\PROJETS\INFO\GODOT\ENGINE\godot_fork\bin\godot.windows.tools.64.exe" "--path" "D:/PROJETS/INFO/GODOT/TESTS/3/lol/OpenGLErrors5" "--editor"
Press <RETURN> to close this window...
Activated GL 3.3 contextOpenGL ES 3.0 Renderer: GeForce 940M/PCIe/SSE2
GLES3: max ubo light: 409
GLES3: max ubo reflections: 455, ubo size: 144
ERROR: ClassDB::bind_methodfi: Couldn't bind method 'set_expand_margin_all' for instance: StyleBoxFlat
   At: core\class_db.cpp:1182
ERROR: ClassDB::bind_methodfi: Method/Function Failed, returning: 0
   At: core\class_db.cpp:1184
ERROR: ClassDB::bind_methodfi: Couldn't bind method 'set_expand_margin_individual' for instance: StyleBoxFlat
   At: core\class_db.cpp:1182
ERROR: ClassDB::bind_methodfi: Method/Function Failed, returning: 0
   At: core\class_db.cpp:1184
mode: spatial
1
WARNING: AudioDriverRtAudio::callback: RtAudio output underflow!
     At: drivers\rtaudio\audio_driver_rtaudio.cpp:57
```

**Comment by RandomShaper** [comment]
#10557 may fix this.

Anyway, if you'd like the debug further, provided Godot manages to create the project, you can add `-path c:\path\to\project -e` to start the debug session right in the editor.

**Comment by Zylann** [comment]
Opening the projects doesn't produce a crash, that only happens as a result of doing it from the Project Manager ^^"

**Comment by RandomShaper** [comment]
I see. Pretty weird.

Please rebuild and retest now #10557 is merged.

**Comment by hpvb** [comment]
Is this still a problem @Zylann

**Comment by Zylann** [comment]
I created a new project with debugger attached, the project opened in a new Godot instance, but after some time the debugger detected a crash of the project manager

![image](https://user-images.githubusercontent.com/1311555/30191029-9ad5fab0-943e-11e7-9be2-a2b7b230a465.png)

**Comment by hpvb** [comment]
Yeah, that should be fixed with #11058

**Comment by Zylann** [comment]
I'll close this then, will reopen if anything similar comes out.
